packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -56,6 +56,7 @@ export const documentUpdateDeleteErrorMapper = (
   match(error.code)
     .with(
       "eServiceNotFound",
+      "eServiceDescriptorNotFound",
```

Comment:
This mapper is used in the `updateDocument` and `deleteDocument` operations. In the first, the error was thrown and not handled. In the second, it was not thrown and not handled. Now it is thrown and handled in both operations.